clippy_lints/src/slice_as_bytes.rs

```diff
@@ -0,0 +1,58 @@
+use clippy_utils::{diagnostics::span_lint_and_sugg, source::snippet_with_applicability, ty::is_type_lang_item};
+use rustc_errors::Applicability::MachineApplicable;
+use rustc_hir::{Expr, ExprKind, LangItem, QPath};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_session::{declare_lint_pass, declare_tool_lint};
+
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for string slices immediantly followed by `as_bytes`.
```

Comment:
```suggestion
    /// Checks for string slices immediatly followed by `as_bytes`.
```

Reply:
i think it's "immediately"  😅 https://dictionary.cambridge.org/dictionary/english/immediately